### DISCUSSION TITLE
Persist JWT in case of docker container restarts

### DISCUSSION THREAD

**Discussion by dionisanchez** [ONLYOFFICE/Docker-DocumentServer#548] [Q&A] [answered]
Hi, I have OnlyOffice running as a Docker container in a Plesk installation making use of Docker extension. I dont know why but my JWT is changing eventually, maybe the container is retarting by himself, mine is created 6 days ago and had it running 4 days long.

Question, can I have JWT persisted so it doesnt change in case of container restart?

Thanks!

**Comment by ShockwaveNN** [comment, marked as answer]
Hi, you should set [JWT_SECRET env var](https://github.com/ONLYOFFICE/Docker-DocumentServer#available-configuration-parameters) for `docker run`

**Reply by dionisanchez** [reply]
That solved my problem, thank you very much.
